packages/design-system/src/styles/_base.scss

```diff
@@ -156,73 +156,74 @@ $breakpoints: (
 
 @mixin heading--5xl {
   @include heading;
-  font-size: var(--font-size-3xl);
+  font-size: var(--typography-heading-5xl__font-size--mobile);
```

Comment:
🍕 in this rule, we're saying the font size starting point is --mobile, with our md media query not having a size suffix